src/main/proto/netflix/managedbatch/job.proto

```diff
@@ -453,6 +483,12 @@ message JobDefinition {
   string applicationName = 7;
 
   JobGroupInfo jobGroupInfo = 8;
+
+  // (Optional) Extra Containers can be specificed to run alongside the main
+  // container in a "pod" (similar to k8s pods). Additional containers
+  // can be specified in this field, and they will be launched together with
+  // the main container, sharing its resources (network/ram/cpu/gpu/etc).
+  repeated BasicContainer extraContainers = 9;
```

Comment:
I don't think this is needed in this CMB proto. This proto is just a **copy** of the internal proto API. I think it's probably a little premature to add it to the internal CMB API unless we think the dust has really settled on the Titus API + implementation and we are ready to roll out the feature in CMB. Whenever we do roll it out in CMB, we should modify the internal version first (along with implementation).

Reply:
Whoops, sorry I didn't realize this file wasn't authoritative.